src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/ResourceUtil.java

```diff
@@ -31,8 +31,10 @@
  * such as TOTAL_THROUGHPUT and SYS_CALL_RATE
  */
 public class ResourceUtil {
-
   // JVM resource
+  public static final Resource HEAP_MAX_SIZE = Resource.newBuilder()
```

Comment:
Do you want to consider *old gen* heap size or total heap size for your configurations? I think [jvm has some issues ](https://e-mc2.net/elasticsearch-garbage-collection-hell) with honoring the young-old split, we should just use total heap size.

Reply:
Created a new ResourceEnum called heap.